ultraplot/internals/rcsetup.py

```diff
@@ -32,6 +32,7 @@
 # NOTE: These are set to True inside __init__.py
 # NOTE: We really cannot delay creation of 'rc' until after registration because
 # colormap creation depends on rc['cmap.lut'] and rc['cmap.listedthresh'].
+# center_levels = _not_none(center_levels, rc["c
```

Comment:
This comment needs to be fixed. IDK what it does or if we should just remove it.

Reply:
ah yes thanks!